README.md

```diff
@@ -9,11 +9,11 @@ solo project N°1
 
 ---
 
-![Example](https://web.archive.org/web/20140301004610/http://www.apple.com/)
+![Example](http://archive.li/UW4oR)
```

Comment:
- [ ] The link its broken on line #12, please remove the `!` sign in the beginning of the line so it looks like this:

`[Example](http://archive.li/UW4oR)`